SpringContainerMS/chart/springcontainerms/templates/deployment.yaml

```diff
@@ -75,26 +81,32 @@ spec:
                 name: "{{ .Values.bpm.anomalySecret }}"
                 key: password
                 optional: true
-          - name: KCSOLUTION_CONTAINERS
+          - name: BPM_EXPIRATION
```

Comment:
This should be pulled from a ConfigMap and not a Secret. It makes more sense since it is not sensitive information and you reference the creation of this element in a ConfigMap in https://github.com/ibm-cloud-architecture/refarch-kc/pull/66/files